src/divider/styled-components.ts

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright (c) Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+import { styled } from '../styles/index';
+import { SIZE } from './constants';
+import type { DividerPropsT } from './types';
+
+export const StyledDivider = styled<'hr', DividerPropsT>('hr', (props) => {
+  const { $theme, $size = SIZE.section } = props;
+  const borderResetStyles = {
+    borderRight: 'none',
+    borderBottom: 'none',
+    borderLeft: 'none',
+  };
+
+  if ($size === SIZE.cell) {
+    return {
+      ...borderResetStyles,
+      borderTop: `1px solid ${$theme.colors.borderOpaque}`,
```

Comment:
We should still probably keep using long hand props only to be safe?

Reply:
Good catch